### ISSUE TITLE
TSDAE with weight_decay?

### ISSUE THREAD

**Issue by jamescalam** [UKPLab/sentence-transformers#1268]
Hey Nils! In your TSDAE paper it says you use AdamW but I can't see weight decay value in the paper, and in [train_tsdae_from_file.py](https://github.com/UKPLab/sentence-transformers/blob/master/examples/unsupervised_learning/TSDAE/train_tsdae_from_file.py) you use a weight_decay of 0.

So my question is, in the paper were you using Adam / AdamW with weight_decay==0? Or did you use weight decay?

Thanks!

**Comment by nreimers** [comment]
@kwang2049 could you answer this question?

**Comment by kwang2049** [comment]
Hi @jamescalam, thanks for your attention!

I have checked my experiment again and the optimizer setting for TSDAE is `AdamW` with `weight_decay=0`. I used 0 because I want the setting to be simple and I did not tune the hyper-parameters, either.

**Comment by jamescalam** [comment]
Okay that makes sense, thanks!
